docker/README

```diff
@@ -1,19 +1,26 @@
 Container image for Zeek
 ========================
 
-This directory contains a minimal container image for Zeek. This image is
-published automatically to [DockerHub](https://hub.docker.com/u/zeekurity) for
+This directory allows building a minimal container image for Zeek. This image is
+published automatically to [DockerHub](https://hub.docker.com/u/zeek) for
```

Comment:
Aren't we also publishing these to the AWS registry?

Reply:
Yes, I've added a small addition there.